newscoop/src/Newscoop/NewscoopBundle/EventListener/LocaleListener.php

```diff
@@ -24,10 +26,16 @@ class LocaleListener
 
     protected $em;
 
-    public function __construct(CacheService $cacheService, EntityManager $em)
-    {
+    protected $session;
+
+    public function __construct(
+        CacheService $cacheService,
+        EntityManager $em,
+        Session $session
+    ) {
         $this->cacheService = $cacheService;
         $this->em = $em;
+        $this->session = $session;
```

Comment:
do you really need to inject session service ? you can get the session from the `$request = $event->getRequest()->getSession()` in `onRequest` method so imo injecting the session service is useless here